blog/2024-06-06-we-are-on-gpt-4o.mdx

```diff
@@ -0,0 +1,27 @@
+---
+slug: we-are-on-chatgpt-4o
+title: We are on GPT 4o 🚀
+description: Tolgee Translator is now backed by GPT-4o, the latest OpenAI's flagship model.
+authors: [jcizmar]
+tags: [chatgpt, tolgee, ai, tolgee-translator]
+---
+
+Great news! 🚀 Tolgee Translator is now backed by GPT-4o, the latest OpenAI's flagship model. It returns better results, and it is much less error-prone.
+
+<!--truncate-->
+
+## It fails less
+The GPT-3-turbo we used before sometimes failed since it could not provide results for some string language
```

Comment:
Our previous model, GPT-3-turbo, sometimes struggled with certain language pairs, leading to translation failures. GPT-4o is more stable and reliable.